### PR TITLE
W-12986516-API-Manager-CLI-copy-editing-KL

### PR DESCRIPTION
**Pull request by kevinzlu-salesforce**

### CODE REVIEW

File: modules/ROOT/pages/_partials/api-mgr.adoc [hunk 1]
```diff
@@ -56,12 +56,12 @@ Besides the default `--help`, general, and authentication flags, this command al
 |Flag |Description
 | `--duration <num>` | Condition occurrence period duration.
 | `--durationUnit <value>` | Condition occurrence period duration unit. +
-Supported values: days, hours, minutes.
+Supported values: `days`, `hours`, `minutes`.
 | `--email <emailAddress>` | Email to send alert notification to. +
 You can pass this flag multiple times to specify multiple emails.
```
Comment (glenn-rodgers-sf) [inline modules/ROOT/pages/_partials/api-mgr.adoc]:
```suggestion
Pass this flag multiple times to specify multiple emails.
```

File: modules/ROOT/pages/_partials/api-mgr.adoc [hunk 2]
```diff
@@ -56,12 +56,12 @@ Besides the default `--help`, general, and authentication flags, this command al
...
 | `--enabled` | Sets whether the alert should be enabled or not. Include the flag to enable it.
```
Comment (glenn-rodgers-sf) [inline modules/ROOT/pages/_partials/api-mgr.adoc]:
```suggestion
| `--enabled` | Sets whether the alert is enabled. Include the flag to enable the alert.
```

File: modules/ROOT/pages/_partials/api-mgr.adoc [hunk 3]
```diff
@@ -56,12 +56,12 @@ Besides the default `--help`, general, and authentication flags, this command al
...
 | `--operator <value>` | Condition operator explaining values relation to threshold. +
-Supported values: gt, lt, eq
+Supported values: `gt`, `lt`, `eq`
```
Comment (glenn-rodgers-sf) [inline modules/ROOT/pages/_partials/api-mgr.adoc]:
```suggestion
Supported values: `gt`, `lt`, `eq`.
```

File: modules/ROOT/pages/_partials/api-mgr.adoc [hunk 4]
```diff
@@ -56,12 +56,12 @@ Besides the default `--help`, general, and authentication flags, this command al
...
 | `--periods <num>` | Number of consecutive periods condition should occur for.
 | `--policyId <num>` | ID of a policy applied to the API instance to trigger `policy-violation` alert type.
```
Comment (glenn-rodgers-sf) [inline modules/ROOT/pages/_partials/api-mgr.adoc]:
```suggestion
| `--policyId <num>` | ID of a policy applied to the API instance that triggers a `policy-violation` alert type.
```

File: modules/ROOT/pages/_partials/api-mgr.adoc (hunk 1 above)
Comment (glenn-rodgers-sf) [reply modules/ROOT/pages/_partials/api-mgr.adoc]: For this, universally change "You can pass" to "Pass."

File: modules/ROOT/pages/_partials/api-mgr.adoc [hunk 5]
```diff
@@ -93,7 +93,7 @@ Besides the default `--help`, general, and authentication flags, this command al
 |===
 |Flag |Description 
 |`--limit <value>` | Number of results to retrieve, default is 10 results
-| `--offset <value>` | Offsets the amount of APIs passed, for example`--offset 3`
+| `--offset <value>` | Offsets the amount of APIs passed, for example, `--offset 3`
 | `--output <value>`| Specifies the response format. + 
 Supported values are `table` (default) and `json`
```
Comment (glenn-rodgers-sf) [inline modules/ROOT/pages/_partials/api-mgr.adoc:98]: I'm honestly not sure if this should be considered a complete sentence without it being "The supported values."

I would ask the editors.

File: modules/ROOT/pages/_partials/api-mgr.adoc [hunk 6]
```diff
@@ -56,18 +56,18 @@ Besides the default `--help`, general, and authentication flags, this command al
 |Flag |Description
 | `--duration <num>` | Condition occurrence period duration.
 | `--durationUnit <value>` | Condition occurrence period duration unit. +
-Supported values: days, hours, minutes.
+Supported values: `days`, `hours`, `minutes`.
```
Comment (glenn-rodgers-sf) [inline modules/ROOT/pages/_partials/api-mgr.adoc]:
```suggestion
Supported values: `days`, `hours`, and `minutes`.
```

Universal

File: modules/ROOT/pages/_partials/api-mgr.adoc [hunk 7]
```diff
@@ -126,7 +126,7 @@ Supported values are `table` (default) and `json`
 > api-mgr:api:change-specification [flags] <apiInstanceId> <assetVersion>
 ----
 
-Changes the asset version for the API instance passed in `<apiInstanceId`,  by choosing a new version from Exchange passed in `<assetVersion>`.
+Changes the asset version for the API instance passed in `<apiInstanceId` by choosing a new version from Exchange passed in `<assetVersion>`.
```
Comment (glenn-rodgers-sf) [inline modules/ROOT/pages/_partials/api-mgr.adoc]:
```suggestion
Changes the asset version for the API instance passed in `<apiInstanceId>` by choosing a new version from Exchange passed in `<assetVersion>`.
```

File: modules/ROOT/pages/_partials/api-mgr.adoc [hunk 8]
```diff
@@ -159,7 +159,7 @@ This command accepts only the default `--help`, general, and authentication flag
 > api-mgr:api:deploy [flags] <apiInstanceId>
 ----
 
-This command deploys the API instance passed in <apiInstanceId> to the deployment target specified using the flags described below. You can deploy any undeployed API using this command regardless of if it was created using the API Manager CLI or API Manger UI.
+This command deploys the API instance passed in <apiInstanceId> to the deployment target specified using the flags described below. Deploy any undeployed API using this command regardless of if it was created using the API Manager CLI or API Manger UI.
```
Comment (glenn-rodgers-sf) [inline modules/ROOT/pages/_partials/api-mgr.adoc]:
```suggestion
This command deploys the API instance passed in '<apiInstanceId>' to the deployment target specified using the flags described below. Deploy any undeployed API using this command regardless of if it was created using the API Manager CLI or API Manger UI.
```

File: modules/ROOT/pages/_partials/api-mgr.adoc [hunk 9]
```diff
@@ -175,15 +175,15 @@ Besides the default `--help`, general, and authentication flags, this command al
 | `--applicationName myMuleApp 643404`
 
 | `--environmentName <name>`
-| Target environment name, used only when deploying APIs from unclassified environments.
+| Target environment name, used only when deploying APIs from unclassified environments
```
Comment (glenn-rodgers-sf) [inline modules/ROOT/pages/_partials/api-mgr.adoc]:
```suggestion
| Target environment name, only for when deploying API instances from unclassified environments
```

File: modules/ROOT/pages/_partials/api-mgr.adoc [hunk 10]
```diff
@@ -233,19 +233,19 @@ For example: `api-mgr:api:download-proxy --gatewayVersion: 4.0.1  643404 /tmp/`
 > api-mgr:api:edit [flag] <apiInstanceId>
 ----
 
-This command edits the API instance passed in `<apiInstanceId>`. If you edit a Flex Gateway API instance that is currently deployed, this command redeploys the edited Flex Gateway API instance. 
+This command edits the API instance passed in `<apiInstanceId>`. Editing a Flex Gateway API instance that is currently deployed will redeploy this.
```
Comment (glenn-rodgers-sf) [inline modules/ROOT/pages/_partials/api-mgr.adoc]:
```suggestion
This command edits the API instance passed in `<apiInstanceId>`. Editing a Flex Gateway API instance that is currently deployed redeploy the API instance.
```

File: modules/ROOT/pages/_partials/api-mgr.adoc [hunk 11]
```diff
@@ -233,19 +233,19 @@ For example: `api-mgr:api:download-proxy --gatewayVersion: 4.0.1  643404 /tmp/`
...
 
 Besides the default `--help`, general, and authentication flags, this command also accepts:
 
 [%header%autowidth.spread,cols="a,a"]
 |===
 |Flag |Description
 | `-f, --isFlex`
-| Indicates whether you are managing this API instance in FlexGateway. +
+| Indicates whether this API instance is being managed in FlexGateway. +
```
Comment (glenn-rodgers-sf) [inline modules/ROOT/pages/_partials/api-mgr.adoc]:
```suggestion
| Indicates whether this is a Flex Gateway API instance. +
```

File: modules/ROOT/pages/_partials/api-mgr.adoc [hunk 12]
```diff
@@ -233,19 +233,19 @@ For example: `api-mgr:api:download-proxy --gatewayVersion: 4.0.1  643404 /tmp/`
...
 Include the flag to enable it.
 
 | `-m, --muleVersion4OrAbove`
-| Indicates whether you are managing this API instance in Mule 4 or above. +
+| Indicates whether this API instance is being managed in Mule 4 or above. +
```
Comment (glenn-rodgers-sf) [inline modules/ROOT/pages/_partials/api-mgr.adoc]:
```suggestion
| Indicates whether this is a Mule 4 API instance. +
```

File: modules/ROOT/pages/_partials/api-mgr.adoc [hunk 13]
```diff
@@ -257,40 +257,41 @@ Include the flag to enable it.
...
+| Consumer endpoint URI.
 
 | `--inboundSecretGroupId <value>` 
-| Inbound secret group ID
+| Inbound secret group ID.
 
 | `--inboundTlsContextId <value>` 
 | Outbound TLS Context ID. +
-You must supply the `--inboundSecretGroupId` of the TLS Context's secret group unless if you are removing a TLS Context. To remove a TLS Context, apply the flag with the following value: `--inboundTlsContextId "null"`.
+Supply the `--inboundSecretGroupId` of the TLS Context's secret group unless the TLS Context is being removed.
```
Comment (glenn-rodgers-sf) [inline modules/ROOT/pages/_partials/api-mgr.adoc]:
```suggestion
Supply the `--inboundSecretGroupId` of the TLS Context's secret group. To remove a TLS Context, apply the flag with the following value: `--inboundTlsContextId "null"`.
```

File: modules/ROOT/pages/_partials/api-mgr.adoc [hunk 14]
```diff
@@ -257,40 +257,41 @@ Include the flag to enable it.
...
+To remove a TLS Context, apply the flag with the following value: `--inboundTlsContextId "null"`.
 
 | `--outboundSecretGroupId <value>`
-| Outbound secret group ID 
+| Outbound secret group ID. 
 
 | `--outboundTlsContextId <value>`
 | Outbound TLS Context ID. +
-You must supply the `--outboundSecretGroupId` of the TLS Context's secret group unless if you are removing a TLS Context. To remove a TLS Context, apply the flag with the following value: `--outboundTlsContextId "null"`.
+Supply the `--outboundSecretGroupId` of the TLS Context's secret group unless a TLS Context is being removed. To remove a TLS Context, apply the flag with the following value: `--outboundTlsContextId "null"`.
```
Comment (glenn-rodgers-sf) [inline modules/ROOT/pages/_partials/api-mgr.adoc]:
```suggestion
Supply the `--outboundSecretGroupId` of the TLS Context's secret group. To remove a TLS Context, apply the flag with the following value: `--outboundTlsContextId "null"`.
```

File: modules/ROOT/pages/_partials/api-mgr.adoc [hunk 15]
```diff
@@ -353,11 +354,11 @@ Besides the default `--help`, general, and authentication flags, this command al
 |Flag |Description
 
 | `-f, --isFlex`
-| Indicates whether you are managing this API instance in Flex Gateway. +
+| Indicates whether this API instance is being managed in Flex Gateway. +
```
Comment (glenn-rodgers-sf) [inline modules/ROOT/pages/_partials/api-mgr.adoc]: See the edits

File: modules/ROOT/pages/_partials/api-mgr.adoc [hunk 16]
```diff
@@ -469,15 +470,15 @@ Besides the default `--help`, general, and authentication flags, this command al
 | Application name
 
 | `--environmentName <name>`
-| Target environment name. +
- Must be provided to redeploy APIs from unclassified environments.
+| Target environment name +
+ Must be provided to redeploy APIs from unclassified environments
```
Comment (glenn-rodgers-sf) [inline modules/ROOT/pages/_partials/api-mgr.adoc]:
```suggestion
Include to redeploy APIs from unclassified environments
```

File: modules/ROOT/pages/_partials/api-mgr.adoc [hunk 17]
```diff
@@ -689,15 +690,15 @@ Besides the default `--help`, general, and authentication flags, this command al
...
 
 | `--description <value>`
-| Tier description
+| Tier description.
 
 | `-l, --limit <value>`
 |Single instance of an SLA tier limit in the form `--limit A,B,C` where:
 
-* `A` is a boolean indicating whether or not this limit should be visible.
+* `A` is a boolean indicating whether this limit should be visible.
```
Comment (glenn-rodgers-sf) [inline modules/ROOT/pages/_partials/api-mgr.adoc]:
```suggestion
* `A` is a boolean indicating whether this limit is visible to the user.
```

Might need to check with engineering on this.

**Review by glenn-rodgers-sf — DISMISSED** [review]

**Review by Crispy-Salesforce — APPROVED** [review]

**Pull request merged**